pkgs/development/tools/knightos/patchrom/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake, asciidoc, libxslt, docbook_xsl }:
+
+
+stdenv.mkDerivation rec {
+  pname = "patchrom";
+
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "patchrom";
+    rev = version;
+    sha256 = "0yc4q7n3k7k6rx3cxq5ddd5r0la8gw1287a74kql6gwkxjq0jmcv";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ asciidoc libxslt.bin docbook_xsl ];
```

Comment:
asciidoc and docbook likely belong to native. Does `libxslt` (without `.bin`) doesn't work?

Reply:
Thanks for the review, yeah without the `.bin` part it works.

Reply:
Oh it breaks NixOS build if I do that.